### DISCUSSION TITLE
Social links in config.yaml return error upon trying to find icons

### DISCUSSION THREAD

**Discussion by Noodulz** [kdevo/osprey-delight#46] [Q&A]
Hi, I set up osprey-delight on my end as a mod and tried running the website using `hugo server`. I got back a big error which seemed to lead back to the social media links in `config.yaml` not being able to find the icons in `./assets/%!s(<nil>)` which in turn does not seem to exist either. Here is the full error message:

```
WARN  image path './assets/%!s(<nil>)' does not exist. please check the 'assets' directory.

Built in 88 ms

Error: error building site: render: failed to render pages: render of "home" failed: "C:\Users\jocel\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\kdevo\osprey-delight\v5@v5.0.2\layouts\index.html:26:7": execute of template failed: template: index.html:26:7: executing "index.html" at <partial "footer.html" .>: error calling partial: "C:\Users\jocel\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\kdevo\osprey-delight\v5@v5.0.2\layouts\partials\footer.html:6:11": execute of template failed: template: partials/footer.html:6:11: executing "partials/footer.html" at <partial "icon" .type>: error calling partial: "C:\Users\jocel\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hugo-mods\icons@v0.0.0-20220508080705-d938a1900b1c\layouts\partials\icon.html:53:19": execute of template failed: template: partials/icon.html:53:19: executing "partials/icon.html" at <partial "icon-resolve" (dict "icon" $icon)>: error calling partial: "C:\Users\jocel\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hugo-mods\icons@v0.0.0-20220508080705-d938a1900b1c\layouts\partials\icon-resolve.html:15:21": execute of template failed: template: partials/icon-resolve.html:15:21: executing "partials/icon-resolve.html" at <resources.Get>: error calling Get: CreateFile C:\Users\jocel\Documents\hugo-portfolio\assets\%!s(<nil>)\linkedin.svg: The filename, directory name, or volume label syntax is incorrect.
```

I tried to resolve this by removing the links under `Social: ` in `config.yaml` which seemed to resolve the errors and I was able to build my site and run it on localhost. Which leads me to believe there's something wrong or missing with the way the social media icons are being retrieved. Is there any current workaround this?

**Comment by kdevo** [comment]
Hi Noodulz,

Looking at the logs of your comment it seems like the path `C:\Users\jocel\Documents\hugo-portfolio\assets\%!s(<nil>)\linkedin.svg` is wrong and it should instead try to get the SVG from the Hugo modules cache.

Looking at `image path './assets/%!s(<nil>)' does not exist` it also seems that there is an empty image `""` path referenced somewhere in your source files.

Few questions for narrowing the issue down:
- Could you post the output of the `hugo version` command? 
- Are you running Hugo as a snap package or in another isolated environment?
- Did you change the `caches.modules` directory?
- Does the reference site https://github.com/hugo-mods/hugo-mods.github.io work for you?

Unrelated to your issue (it seems): While trying to reproduce your issue, I found that the `exampleSite` of the theme is broken atm. It doesn't find the `stackoverflow` icon. Seems like Font Awesome changed the name of it in a recent update. Will fix this the coming days. #43

**Comment by kdevo** [comment]
Hey @Noodulz, I could not reproduce the issue, so I've converted this to a discussion.
Please let me know if your issue is resolved already or if you have any more details as described in my previous comment.
